css/custom.css

```diff
@@ -59,6 +59,18 @@ i.fa.fa-times:hover {
   color: red;
 }
 
+.languagepicker.roundborders.large .select-your-language:hover {
+  background-color: #80BE62 ;
+}
+
+.languagepicker.roundborders.large .hover-black:hover {
+  background-color: rgba(0,0,0,0.2);
+}
+
+.right.hide-on-med-and-down li:hover{
```

Comment:
This is not covered by a class.

Reply:
For this I think class is not needed as in future also if we add more menu tabs it will be in "li".